app/scripts/modules/app/index.html

```diff
@@ -0,0 +1,268 @@
+                  id="Shape"
+                  fill="url(#linearGradient-2)"
+                  style="mix-blend-mode: multiply"
+                ></path>
+              </g>
+            </g>
+          </g>
+        </g>
+      </svg>
+
+      <h1 id="welcome-message" ng-if="!authenticating">Hello!</h1>
+      <h1 style="display: none" ng-style="{display: authenticating ? '' : 'none'}">Authenticating...</h1>
+
+      <div class="bars">
+        <div class="bar"></div>
+        <div class="bar"></div>
+        <div class="bar"></div>
+        <div class="bar"></div>
+        <div class="bar"></div>
+      </div>
+    </div>
+
+    <!-- Replaces "Welcome!" message periodically with a random quote -->
+    <script>
+      (function () {
+        var messageOptions = [
+          "If I'm not back in five minutes... just wait longer.", // Ace Ventura: Pet Detective
+          "And that's why you always leave a note.", // Arrested Development
+          'Jim never has a second cup of coffee at home.', // Airplane!
+          'This is WZAZ in Chicago, where disco lives forever.', // Airplane!
+          'Carpe dentum. Seize the teeth.', // Mrs. Doubtfire
+          'Ah! Norman Bates!', // Airplane! (it was a really good movie)
+          'I am the walrus.', // The Big Lebowski
+          "Do you want ants? Because that's how you get ants!", // Archer
+          'Kids you can literally fly!', // Key and Peele
+          "I'd like to add you to my professional network",
+          'Reticulating splines', // Simcity 2000
+          'They mostly come at night. Mostly.', // Aliens
+        ];
+
+        if (Math.random() < 0.02) {
+          var idx = Math.floor(Math.random() * messageOptions.length);
+          document.querySelector('#welcome-message').innerHTML = messageOptions[idx];
+        }
+      })();
+    </script>
+
+    <script>
+      window.addEventListener('load', () => {
+        const settings = window.spinnakerSettings;
+        if (settings && settings.analytics && (settings.analytics.ga || settings.analytics.customConfig)) {
+          (function (i, s, o, g, r, a, m) {
+            i['GoogleAnalyticsObject'] = r;
+            (i[r] =
+              i[r] ||
+              function () {
+                (i[r].q = i[r].q || []).push(arguments);
+              }),
+              (i[r].l = 1 * new Date());
+            (a = s.createElement(o)), (m = s.getElementsByTagName(o)[0]);
+            a.async = 1;
+            a.src = g;
+            m.parentNode.insertBefore(a, m);
+          })(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');
+        }
+      });
+    </script>
+
+    <!-- Actually injects our application -->
+    <spinnaker ng-if="!authenticating"></spinnaker>
+
+    <script src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
+    <script type="module" src="./src/settings.js"></script>
```

Comment:
Looking at the webpack config - What about `settings-local`? Do we need it?

Reply:
With vitejs, `.env.local` file will be used to provide local overrides.